src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29">
+      <div class="calendar__day  calendar__day--start-day_fri"></div>
```

Comment:
start-day should be a modifier for the calendar block

```suggestion
    <div class="calendar calendar--month-length__29 calendar--start-day_fri">
      <div class="calendar__day "></div>
```